src/components/about/InfoLink.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Link from "next/link";
+import { Icon, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
+      textAlign: "center",
+      marginBottom: theme.spacing(8)
+    },
+    linkText: {
+      textDecoration: "underline",
+      color: "inherit"
+    }
+  };
+});
+
+export default function InfoLink({ data }) {
+  const classes = useStyles();
+
+  //use <Link> component only if link href is also on the climateconnect.earth domain
+  if (data.internal)
+    return (
+      <Typography variant="h4" color="primary" className={classes.root}>
+        <Link href={data.href}>
+          <a className={classes.linkText} target="_blank" rel="noopener noreferrer">
```

Comment:
If it's an internal link, should it open in a new window like this?

Reply:
You're right, it shouldn't. Changed it.